webcompat/static/css/development/components/style-variable.css

```diff
@@ -0,0 +1,57 @@
+/*----------Variable----------*/
+
+/* base */
+:root {
+  --base-color : #333;
+  --base-background: #fff;
+  --base-fontFamily: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+  --base-fontSize: 1rem;
+  --base-minWidth:	20rem;
+  --base-maxWidth: 120rem;
+  --base-widthContent : 73.75rem;
+  --base-colorDark: #ffc900; /*Dark Yellow*/
+  --base-variantColorDark: #404040; /* Dark Gray */
+  --base-variantColorLight: #c2c2c2; /* Dark Light */
+  --base-stateNew: #ff8364; /* light orange */
+  --base-stateNeedsDiagnosis: #efce82;  /* light brown */
+  --base-stateNeedsContact : #b9edce ; /* light green */
+  --base-stateReady: #7abce1; /* light blue*/
+  --base-stateSitewait: #b0daf0; /* light blue */
+  --base-stateClose: #c5d1d5; /* light gray */
+}
+
+/* link */
+:root {
+  --Link-color: #000;
+  --Link-colorHover: var(--Link-color);
+  --Button-disabled-opacity : 0.5; /* button opacity disabled*/
+}
+
+/* Custom media queries */
+@custom-media --wc-max330px all and (max-width: 20.62em);
+@custom-media --wc-min330px all and (min-width: 20.63em);
+
+@custom-media --wc-max375px all and (max-width: 23.43em);
+@custom-media --wc-min375px all and (min-width: 23.44em);
+
+
+@custom-media --wc-max400px all and (max-width: 25em);
+@custom-media --wc-min400px all and (min-width: 25.01em);
+
+
+@custom-media --wc-max450px all and (max-width: 28.12em);
+@custom-media --wc-min450px all and (min-width: 28.13em);
+
+@custom-media --wc-max600px all and (max-width: 37.5em);
+@custom-media --wc-min600px all and (min-width: 37.51em);
+
+@custom-media --wc-max770px all and (max-width: 48.12em);
+@custom-media --wc-min770px all and (min-width: 48.13em);
+
+@custom-media --wc-max950px all and (max-width: 59.37em);
+@custom-media --wc-min950px all and (min-width: 59.38em);
+
+@custom-media --wc-max990px all and (max-width: 61.87em);
+@custom-media --wc-min990px all and (min-width: 61.88em);
+
+@custom-media --viewport-1180px all and (max-width: 73.75em);
```

Comment:
Yeah I didn't replace yet all custom-media. I'm not a big fan of the new name I choose. We need something simple, semantic and easy to remember.